services/sensu/service.go

```diff
@@ -133,9 +136,10 @@ func (s *Service) prepareData(name, output string, level alert.Level) (*net.TCPA
 
 	postData := make(map[string]interface{})
 	postData["name"] = name
-	postData["source"] = c.Source
+	postData["source"] = hostname
 	postData["output"] = output
 	postData["status"] = status
+	postData["auto_resolve"] = false
```

Comment:
If a node has a crit alert then a ok alert, the node is always in crit. In our case, this allow to find Flapping on network equipment.